services/species.service.js

```diff
@@ -0,0 +1,36 @@
+const { v4: uuid } = require('uuid');
+const SpeciesModel = require('../models/species.schema');
+
+const getAllSpecies = async () => {
+  return SpeciesModel.find({});
+};
+
+const changeListing = async (speciesId) => {
+  const changedListing = !SpeciesModel.findById(speciesId).isAssigned;
```

Comment:
We need some way to edit species.isEndangered. Also, it might be better to specify true/false from the request input rather than toggling.